src/pages/Profile.js

```diff
@@ -465,48 +573,84 @@ const Profile = ({ queryParams }) => {
                 }, ['user guide.'])
               ])
             ]),
-            div({ style: { margin: '1rem' } }, [proxyGroup]),
+            div({ style: { margin: '1rem' } }, proxyGroup ? proxyGroup : 'Loading...')
+          ])
+        ]),
 
-            sectionTitle('Location'),
+        sectionTitle('What is your area of research?'),
 
-            line([
-              textField('programLocationCity', 'City'),
-              textField('programLocationState', 'State')
-            ]),
-            line([
-              textField('programLocationCountry', 'Country')
-            ]),
+        p('Check all that apply.'),
 
-            sectionTitle('Account Notifications'),
-
-            checkbox('notifications/GroupAccessRequestNotification', 'Group Access Requested'),
-            checkbox('notifications/WorkspaceAddedNotification', 'Workspace Access Added'),
-            checkbox('notifications/WorkspaceRemovedNotification', 'Workspace Access Removed'),
-
-            h(ButtonPrimary, {
-              style: { marginTop: '3rem' },
-              onClick: _.flow(
-                Utils.withBusyState(setSaving),
-                withErrorReporting('Error saving profile')
-              )(async () => {
-                const [prefsData, profileData] = _.over([_.pickBy, _.omitBy])((_v, k) => _.startsWith('notifications/', k), profileInfo)
-                await Promise.all([
-                  Ajax().User.profile.set(_.pickBy(_.identity, profileData)),
-                  Ajax().User.profile.setPreferences(prefsData)
-                ])
-                await refreshTerraProfile()
-              }),
-              disabled: !!errors,
-              tooltip: !!errors && 'Please fill out all required fields'
-            }, ['Save Profile'])
+        line([
```

Comment:
minor/nit - maybe remove the top margin? There's a lot of space between `Check all that apply` and the checkboxes
<img width="374" alt="Screen Shot 2022-09-07 at 2 12 28 PM" src="https://user-images.githubusercontent.com/5375000/188952184-fb7c2c06-b521-472f-8ee0-cb9359a0ba5e.png">

Reply:
Reduced margin by half in 95398d07aa7457ffcacaa1593683672a79301a6b